inst/extdata/config_schema.json

```diff
@@ -0,0 +1,191 @@
+                    "format": "date"
+                },
+                "disease": {
+                    "type": "string"
+                },
+                "geo_value": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "geo_type": {
+                    "type": "string"
+                },
+                "parameters": {
+                    "$ref": "#/definitions/Parameters"
+                },
+                "data": {
+                    "$ref": "#/definitions/Data"
+                },
+                "seed": {
+                    "type": "integer"
+                },
+                "horizon": {
+                    "type": "integer"
+                },
+                "priors": {
+                    "$ref": "#/definitions/Priors"
+                },
+                "sampler_opts": {
+                    "$ref": "#/definitions/SamplerOpts"
+                }
+            },
+            "required": [
+                "as_of_date",
+                "data",
+                "disease",
+                "geo_type",
+                "geo_value",
+                "horizon",
+                "job_id",
+                "parameters",
+                "priors",
+                "sampler_opts",
+                "seed",
+                "task_id"
+            ],
+            "title": "Epinow2"
+        },
+        "Data": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["null", "string"]
+                },
+                "report_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                },
+                "reference_date": {
+                    "type": "array",
+                    "items": {
+                        "type": "string",
+                        "format": "date"
+                    }
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path",
+                "reference_date",
+                "report_date"
+            ],
+            "title": "Data"
+        },
+        "Parameters": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "path": {
+                    "type": "string"
+                },
+                "blob_storage_container": {
+                    "type": ["string", "null"]
+                }
+            },
+            "required": [
+                "blob_storage_container",
+                "path"
+            ],
+            "title": "Parameters"
+        },
+        "Priors": {
```

Comment:
Would you always needs to pass in these arguments or is this specific to Epinow2? I think there might be other packages where you would handle specifying priors differently (e.g. in the ww package we're developing, priors are lumped in with parameters...)

Reply:
My assumption is that you'd have to edit this in another version of the repo to enforce that schema, but I think the framework is adaptable!

Reply:
Given that something like `fetch_config` is not `EpiNow2` specific there is some argument it shouldn't be in this package. It seems like if we are to have another package `cfa-newpackage-pipeline` then it'll also need things function. So the schema is `EpiNow2` specific but the surrounding functions are not